exercises/ex3/README.md

```diff
@@ -40,7 +40,13 @@ Currently, buyers in the web shop can calculate a promotion discount by clicking
 <br>![](/exercises/ex3/images/maven_plugin.png)
 
 
-5. Run the client library generator by executing the following commands in your terminal.
+5. In order for the plugin to work, you need install its dependencies first by running:
+```
+mvn clean install
```

Comment:
Not sure if the "clean" is really needed, but since build time should be fast it doesn't hurt to leave it in.